sdk/purview/purview-scanning-rest/package.json

```diff
@@ -115,8 +117,7 @@
       "browser",
       "react-native"
     ],
-    "selfLink": false,
-    "project": "./tsconfig.src.json"
```

Comment:
is this line still needed? `"project": "./tsconfig.src.json"`

Reply:
this line is moved in https://github.com/Azure/azure-sdk-for-js/pull/32461/files/0fec6615abb8dfcbdf593805c4732a6872aece3c#diff-ce96f9139dfc90feb428a5cfc2d67b6af8476c3c3e1a5362eefa45bd800d7152R107

Reply:
if it's not necessary, maybe migration tool should handle it?

Reply:
or you mean this should not be deleted? this line is still there, it's moved into line 107.

Reply:
ah got it!